app/models/admin_authorization.rb

```diff
@@ -0,0 +1,13 @@
+class AdminAuthorization < ActiveAdmin::AuthorizationAdapter
+
+  def authorized?(action, subject = nil)
+    if Thread.current[:user].present?
```

Comment:
Why is this done using `Thread` and not the sessions?

Reply:
I wasn't able to access the session from within this model, and I'm not ever explicitly calling it, this is called before every active admin action to check permission, so I couldn't pass extra options to it. Is there a better way?

Reply:
Have you tried this way?
https://activeadmin.info/13-authorization-adapter.html

Reply:
This is what I used, but for me the `user` method always returns `nil`, have you ever used it?